engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetWireframe.java

```diff
@@ -33,11 +35,21 @@ public SetWireframe(boolean enabled) {
         this.enabled = enabled;
     }
 
+    public SetWireframe() {
```

Comment:
I'd remove this: it's not necessary for what I can see. Just do:

private static SetWireframe defaultInstance = new SetWireframe(DEFAULT_VALUE);